proto/kvrpcpb.proto

```diff
@@ -993,3 +993,23 @@ message GetLockWaitInfoResponse {
     string error = 2;
     repeated deadlock.WaitForEntry entries = 3;
 }
+
+message RawCoprocessorRequest {
+    kvrpcpb.Context context = 1;
+
+    string copr_name = 2;
+    // Coprorcessor version constraint according to the SEMVER definition.
+    string copr_version_req = 3;
+
+    repeated KeyRange ranges = 4;
```

Comment:
Can we move this field to the end of the message struct in order to keep compatibility with previous proto?

Reply:
Your concern is reasonable, but since the RPC has breakingly changed, it will not be worse for here.